pkg/admissioncontroller/webhooks/admission/auditpolicy/admission.go

```diff
@@ -288,3 +317,12 @@ func hasAuditPolicy(shoot *gardencorev1beta1.Shoot) bool {
 		apiServerConfig.AuditConfig.AuditPolicy.ConfigMapRef != nil &&
 		len(apiServerConfig.AuditConfig.AuditPolicy.ConfigMapRef.Name) != 0
 }
+
+func hasAuditPolicyInternal(shoot *gardencore.Shoot) bool {
```

Comment:
I would prefer to have this method only once. You can use `gardencorev1beta1.Convert_v1beta1_ShootList_To_core_ShootList` in the `admitConfigMap` method to convert the v1beta1 shoot list to the internal version. But this is only personal opinion.

Reply:
Yeah, I was thinking about doing some conversion.
Though, I chose not to do so, because having a func for both API versions seems much more efficient. Converting all shoots of a namespace everytime one of the auditpolicy configmaps is changed seemed like quite the overhead to me.

Reply:
Thanks!